content/post/commonifying-llvm-in-nixpkgs/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Commonifying LLVM in Nixpkgs
+description: How I was able to reduce complexity and technical debt in Nixpkgs for LLVM.
+date: 2024-06-08 18:16:00 +0800
+---
+
+In programming, technical debt is a real thing. It often occurs with legacy code or
+things which are not maintained often. Recently, I picked up maintainership in Nixpkgs
```

Comment:
"I picked up maintainership" sounds like you can just buy it at a store (maybe you can IDK), perhaps "I became a maintainer for LLVM in Nixpkgs" would work better

Reply:
I've reworded that, how is it now?